app/controllers/policies_controller.rb

```diff
@@ -22,9 +24,23 @@ def create
     else
       redirect_to  new_policy_path
     end
+  end
 
+ 	def update
+    if @policy.update(policies_params)
+      redirect_to policies_path
+    else
+      redirect_to new_policy_path
+    end
   end
 
+  def destroy
+    if @policy.destroy
+      redirect_to policies_path
+    else
+      redirect_to new_policy_path
```

Comment:
@GauravMakwana-BTC : why it is redirecting to new policy path in case of failure?